docs/user-guide/creating-application/deployment-template/rollout-deployment.md

```diff
@@ -208,10 +216,12 @@ ingress:
       - host: example1.com
         paths:
             - /example
+        pathType:
```

Comment:
Put this before paths field to remove confusion and what about the value of pathType:?
Example:

```
  hosts:
      - host: example1.com
        pathType: "ImplementationSpecific"
        paths:
            - /example
```

Reply:
Repeat all all places

Reply:
done